src/tutorial/TutorialEventArgs.cs

```diff
@@ -123,3 +123,13 @@ public MicrobeColonyEventArgs(MicrobeColony? colony)
 
     public bool HasColony => Colony != null;
 }
+
+public class EnergyBalanceEventArgs : TutorialEventArgs
+{
+    public EnergyBalanceEventArgs(EnergyBalanceInfo? energyBalanceInfo)
```

Comment:
I think the balance should not be nullable here. To me it doesn't make sense to send an event saying "this maybe changed but who knows what it is as it is just null now".

Reply:
I see. Will make it not nullable then. I believe I had done it because multiple other event args used nullable too, but I see that it is not necessary now.

Reply:
Many of the other event types need to be able to signal data missing. I don't think that is necessary in this case. For example the find glucose tutorial can be signaled that the glucose it was told about is gone and there's no new glucose anywhere. The tutorial reacts to that by temporarily disabling itself.